client/me/help/help-contact/index.jsx

```diff
@@ -494,7 +503,20 @@ const HelpContact = React.createClass( {
 		// Hide the olark widget in the bottom right corner.
 		olarkActions.hideBox();
 
-		return <HelpContactForm { ...contactFormProps } />;
+		const shouldShowTicketRequestErrorNotice = ! showChatVariation && null !== ticketSupportRequestError;
+
+		return (
+			<div>
+				{ shouldShowTicketRequestErrorNotice &&
+					<Notice
+						status='is-error'
+						text={ this.translate( 'An error occurred while requesting for email support settings. Please try again later.' ) }
```

Comment:
> An error occurred while requesting ~~for~~ email support settings. Please try again later.

I'm not a big fan of the word 'error', it sounds very final. How about something like:

> We had trouble loading the support information for your account. Please check your internet connection and reload the page, or try again later.